sdk/communication/Azure.Communication.CallingServer/src/Models/CallingServerErrorResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.CallingServer
+{
+    [CodeGenModel("CommunicationErrorResponse")]
+    public partial class CallingServerErrorResponse
```

Comment:
This class is not currently used, correct? If you plan to convert low level exceptions to this, you should only add this class when the conversion code is added. Otherwise, it is confusing

Reply:
@navali-msft could you please remove

Reply:
Removed the public version. Keep the one generated by sdk.